src/Components/Navbar/Components/NavbarMenu/NavbarMenu.scss

```
@@ -0,0 +1,109 @@
+.NavbarMenu {
+  width: 100%;
+
+  ul.gnbMenu {
+    display: flex;
```

```suggestion
.NavbarMenu {
  .gnbMenu {
    display: flex;
```
- 최상위 태그 block 요소인 div 태그인데 가로 100% 필요한가요?
- 이미 .gnbMenu라는 클래스 셀렉터로 구분되니까 ul은 안써줘도 될거 같습니다.